opslevel/resource_opslevel_integration_aws.go

```diff
@@ -175,12 +188,18 @@ func (r *IntegrationAwsResource) Update(ctx context.Context, req resource.Update
 		resp.Diagnostics.Append(diags...)
 		return
 	}
+	regionOverride, diags := ListValueToStringSlice(ctx, planModel.RegionOverride)
+	if diags != nil && diags.HasError() {
+		resp.Diagnostics.Append(diags...)
+		return
+	}
 	input := opslevel.AWSIntegrationInput{
 		Name:                 opslevel.RefOf(planModel.Name.ValueString()),
 		IAMRole:              opslevel.RefOf(planModel.IamRole.ValueString()),
 		ExternalID:           opslevel.RefOf(planModel.ExternalID.ValueString()),
 		OwnershipTagOverride: opslevel.RefOf(planModel.OwnershipTagOverrides.ValueBool()),
 		OwnershipTagKeys:     ownershipTagKeys,
+		RegionOverride:       &regionOverride,
```

Comment:
What happens if the customer doesn't have this in their terraform config but has some values set via the API?

I would suspect this wipes out their settings.  If thats true we either need to require the end user to set this via the schema OR we only put it into the update if they have actually specified it. (unless `ListValueToStringSlice` somehow does that correctly)

I would like to make sure this avenue is tested